### DISCUSSION TITLE
Pretraining resources of ESM2

### DISCUSSION THREAD

**Discussion by GanjinZero** [facebookresearch/esm#414] [Q&A] [answered]
I am interesting to know the pretraining resources for different sizes of ESM2 (GPU count and training time). Thank you.

**Comment by halilakin** [comment, marked as answer]
ESM2 700M had 1.6e22 TFLOPs compute
512 GPUs * 45TFLOPs per sec * (60*60*24) * 8 days on V100s

ESM2 3B had 6e22 TFLOPs compute (~4x over 700M)
512 GPUs * 45TFLOPs per sec * (60*60*24) * 30 days on V100s

ESM2 15B had 1.2e23 TFLOPs compute (2x over 3B)
